extensions/email/src/main/java/org/mule/extension/email/internal/exception/EmailConnectionException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.internal.exception;
+
+import org.mule.runtime.api.connection.ConnectionException;
+
+/**
+ * Is the base {@link ConnectionException} for the Email Connector,
+ *
+ * @since 4.0
+ */
+public class EmailConnectionException extends ConnectionException
```

Comment:
is just a custom more descriptive EmailConnectionException, instead of using the ConnectionExceptionProvided by the sdk.